main.tf

```diff
@@ -51,15 +55,24 @@ resource "aws_lambda_function" "lambda_application" {
     }
   }
 
-  tags = merge({ Name = format("%s-%s", var.application_name, each.value.name) }, { "Lambda Application" = var.application_name }, var.tags)
+  tags = merge({ Name = format("%s-%s", var.application_name, each.value.name) }, { "Lambda Application" = var.application_name }, { "version" = var.application_version }, var.tags)
+}
+
+resource "aws_lambda_alias" "lambda_application_alias" {
+  for_each = var.lambda_functions_config
+
+  name             = var.alias_name
+  description      = "Alias that points to current lambda application version"
```

Comment:
If we have a variable for the alias name, it would make sense we provide a variable for the alias description. Out of interest - I'm not sure I like the alias configuration being coupled to the lambda function config but it will do for now